packages/tasit-contracts/test/LANDRegistry.js

```diff
@@ -0,0 +1,19 @@
+// Note:
+// Should we use ethers instead of web3js?
```

Comment:
If we're going to use Truffle, I think it makes sense to use their testing framework, which could possibly switch to ethers behind the scenes at some point.

Reply:
Resolving this because you gave it a thumbs up. Feel free to delete the note though at some point.